### PR TITLE
test: exam timetable agent submission

### PR DESCRIPTION
**Pull request by JacobRMullally**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js [hunk 1]
```diff
@@ -44,24 +56,49 @@ export class PO_ExaminationTimetable {
...
-	checkAnswers(expectedAnswers) {
+	checkAnswersFirstPage(expectedAnswers) {
 		this.getAllAnswers().then((answers) => {
 			const myAnswers = [];
 			cy.wrap(answers)
 				.each((ans) => myAnswers.push(ans.text().trim()))
 				.then(() => {
-					console.log(myAnswers);
 					myAnswers.shift();
+					console.log({ expectedAnswers, myAnswers });
```
Comment (Yakitrak) [inline packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js:82]: Can we remove this please

File: packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js [hunk 2]
```diff
@@ -44,24 +56,49 @@ export class PO_ExaminationTimetable {
...
 
-	checkAnswersFinal(expectedAnswersFinal) {
+	checkAnswersSecondtPage(expectedAnswers) {
+		this.getAllAnswers().then((answers) => {
+			const myAnswers = [];
+			cy.wrap(answers)
+				.each((ans) => myAnswers.push(ans.text().trim()))
+				.then(() => {
+					myAnswers.pop();
+					console.log({ expectedAnswers, myAnswers });
```
Comment (Yakitrak) [inline packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js:95]: And this please

File: packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js [hunk 3]
```diff
@@ -44,24 +56,49 @@ export class PO_ExaminationTimetable {
...
+					expect(expectedAnswers).to.deep.eq(myAnswers);
+				});
+		});
+	}
+
+	/*checkAnswersFinal(expectedAnswersFinal) {
```
Comment (Yakitrak) [inline packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js:101]: Is this supposed to be removed?

**Review by Yakitrak — CHANGES_REQUESTED** [review]
Left a few comments to tidy up comments and console logs. Also the checks are failing - try pulling main into your branch.

File: packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js (hunk 1 above)
Comment (JacobRMullally) [reply packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js:82]: Have removed this

File: packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js (hunk 2 above)
Comment (JacobRMullally) [reply packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js:95]: have removed this

File: packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js (hunk 3 above)
Comment (JacobRMullally) [reply packages/e2e_tests/cypress/pageObject/Examination-TimeTable/PO_ExaminationTimetable.js:101]: Have removed this

**Pull request closed without merging**